include/drivers/pmic/axp803.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_PMIC_AXP803_H
+#define DRIVERS_PMIC_AXP803_H
+
+#include <pmic.h>
+
+#define AXP803_I2C_ADDRESS 0x34
```

Comment:
I don't like having this defined twice. Maybe remove the regulator one, only switching from number to macro after adding this driver?

Reply:
Agreed. Done.